Pages/FoodCustomer/Details.cshtml.cs

```diff
@@ -0,0 +1,58 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using System.Xml.Linq;
+using FoodKept.Models;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using Microsoft.EntityFrameworkCore;
+
+namespace FoodKept.Pages.FoodCustomer
+{
+    [Authorize(Roles = "Admin, Restaurant, Customer")]
+    public class DetailsModel : PageModel
+    {
+        private readonly FoodKept.Data.ShopContext _context;
+
+        public DetailsModel(FoodKept.Data.ShopContext context)
+        {
+            _context = context;
+        }
+
+        public Food Food { get; set; }
+        public static double Lat { get; set; }
+        public static double Lng { get; set; }
+
```

Comment:
why these are static

Reply:
Accessed in script so need to be static or an error is shown